db/migrations/20220220181650_add_timestamp.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param { import("knex").Knex } knex
```

Comment:
Good idea to add timestamp to books table, should also probably add to favorites